src/pre_process/m_compute_levelset.fpp

```diff
@@ -344,6 +347,120 @@ contains
 
     end subroutine s_compute_rectangle_levelset
 
+    subroutine s_compute_cuboid_levelset(levelset, levelset_norm, ib_patch_id)
```

Comment:
can we change all the `s_compute_XXXXXX` routines here to `s_XXXXXX`. I suppose the `compute` part is implied.